README.md

```diff
@@ -10,9 +8,19 @@ Shunter is a [Node.js][node] application built to read JSON and translate it int
 [![Dependencies][shield-dependencies]][info-dependencies]
 [![LGPL-3.0 licensed][shield-license]][info-license]
 
-![Shunter as a proxy](docs/diagram.png)
+Shunter is a [Node.js][node] application built to read JSON and translate it into HTML.
+
+It helps you create a decoupled front end which can serve traffic from one or more backend applications - great for use in multi-language, multi-disciplinary teams or just to make your project more flexible and future-proofed.
```

Comment:
I'd be tempted to move the badges below this block, looking at the rendered markdown it looks like they have priority over the description.